src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorSingleCommentSeparatedFromPackage.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Config = default
+ * allowNoEmptyLineBetweenFields = false
```

Comment:
@rnveach , @pbludov , .... it is not really a blocker, but probably we should reference here even `tokens` value, so here will be `tokens = [PACKAGE_DEF]`.
It will be even more easy to read Input, and original ideal of such inlined configuration was to eventually create logic in UT that simply read Input file , read  config inside it, and match `\\warn` with actual violations from execution.

Reply:
@rnveach , please confirm this request.

Reply:
@strkkk, as code changes are not avoidable please add `tokens`.
It suppose to be long list and to avoid problems with LineLength we can use same trick as at https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L183

```
tokens = PACKAGE_DEF
tokens =  ... All others tokens  ...
tokens =  ... All others tokens 2 ...
```
Moving `tokens = PACKAGE_DEF` on separate line will give a signal what token is critical .

Reply:
Yes I saw the request.

Reply:
done.

Reply:
@strkkk , I do not see changes that I did for `tokens` and pushed to your remote, please restore them.
Looks like you overwrote my changes.

Reply:
@romani I didn't see that there were your changes in this branch. Usually I do force push for all my changes to keep everything in one commit, so your changes were deleted.

Reply:
Sad, I tried to help ...
Do you want me to not push changes to your PRs in future ?

Reply:
I have local copy of my changes ...
I pushed them here - https://github.com/romani/checkstyle/commit/feec95fa1b43fdd5ea2412e972d0561090f13783
please put them in your branch or add such changes yourself.

Reply:
> Do you want me to not push changes to your PRs in future ?

I am ok with it, just let me know in advance, I will refrain from force pushes

Reply:
ok, I will do this more clearly in future than I did at https://github.com/checkstyle/checkstyle/pull/7251#discussion_r357768289